sunpy/visualization/animator/wcs.py

```diff
@@ -299,10 +303,30 @@ def plot_start_image_2d(self, ax):
 
         return im
 
+    def update_bounds(self):
+        """
+        Update vmin/vmax when set to 'auto'.
+        """
+        print(self.imshow_kwargs)
+        if 'vmax' in self.imshow_kwargs and self.imshow_kwargs['vmax'] == 'auto':
+            # set clip interval to 90% of maximum value
+            clip_interval = (1, 90)*u.percent
+            clip_percentages = clip_interval.to('%').value
```

Comment:
Can these two lines not be combined?

Reply:
Do you want them to be in one single line? Or do you want it split further? I'm a bit confused

Reply:
Can they not be one single line?